state-sync/diem-data-client/src/diemnet/mod.rs

```diff
@@ -78,18 +79,23 @@ pub struct DiemNetDataClient {
 
 impl DiemNetDataClient {
     pub fn new(
-        config: StorageServiceConfig,
+        storage_service_config: StorageServiceConfig,
```

Comment:
super nitty: if it were me, I'd take in a `data_client_config` before a `storage_service_config` because it seems more natural, i.e., the `data_client_config` is directly related to the `DiemNetDataClient`. But, feel free to ignore this comment :P

Reply:
Put `data_client_config` before `storage_service_config`.